### PR TITLE
Feat/setup

### PR DESCRIPTION
**Pull request by sumsumcity**
RaffisVersion drawio

### CODE REVIEW

**Review by wurstbrot — CHANGES_REQUESTED** [review]
`target` folder doesn't belong into the repo. Everyone should create it on their own.
Please use `switch`.

File: src/main/java/com/analyzer/App.java [hunk 1]
```diff
@@ -1,32 +1,21 @@
...
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
+import java.util.Map;
 
+public class App {
+    public static void main(String[] args) {
+        // TODO: Scanner which gets all yaml files the config.yaml file --> put it in utils
+        String yamlFilePath = "./activity_definitions/configuration.yaml";
```
Comment (wurstbrot) [inline src/main/java/com/analyzer/App.java:8]: A path should be in a variable (ENV or start parameter) to be changed by calling the app

File: activity_definitions/configuration.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+conduction of simple threat modeling on a technical level:
```
Comment (wurstbrot) [inline activity_definitions/configuration.yaml:1]: A product owner shouldn't be able to change threshold or level.
See also https://github.com/devsecopsmaturitymodel/metricAnalyzer/pull/11/files .

While initial developing, it is ok to do it. Soon it should be changed

File: src/main/java/com/analyzer/ActivityDirector.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        String level = (String) data.get("level");
+
+        builder = builder
+            .setActivityName(activityName)
+            .setLevel(level);
+
+        // Add Components in the builder
+        ArrayList arr = (ArrayList) data.get("components");
+        for (Object element : arr) {
+            if (element instanceof LinkedHashMap) {
```
[the remark this answers is not in the document]
Comment (sumsumcity) [reply src/main/java/com/analyzer/ActivityDirector.java:40]: Done in the new commit

File: src/main/java/com/analyzer/App.java (hunk 1 above)
Comment (sumsumcity) [reply src/main/java/com/analyzer/App.java:8]: This is in #7 

I will implement this when I am doing the Scanner class

File: activity_definitions/configuration.yaml (hunk 2 above)
Comment (sumsumcity) [reply activity_definitions/configuration.yaml:1]: The configuration.yaml is intended to be set only once by a member of the security team. This was my interpretation of the setup.

**Comment by wurstbrot** [comment]
What do you think about the package `org.owasp.dsomm.metricCA.analyzer`?

**Pull request merged**